alliance/apps/backlog/views/github_webhook_view.py

```diff
@@ -31,6 +31,7 @@ def post(self, request):
         signature_ver = self.verify_signature(request)
         if not signature_ver:
             event = request.META.get(self.meta_key_formatter('X-GitHub-Event'))
+            response = None
```

Comment:
Oh yes - this was one of the choices I made when I rebased - wrong choice I guess. OK, at least it's not something that just was "lost" which was worrying me.